samples/inputs/textarea/overview/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react-input-overview",
+  "description": "This project provides example of Input Overview using Infragistics React components",
+  "author": "Infragistics",
+  "homepage": ".",
+  "version": "1.4.0",
+  "private": true,
+  "scripts": {
+    "start": "set PORT=4200 && react-scripts --max_old_space_size=10240 start",
+    "build": "react-scripts --max_old_space_size=10240 build ",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "dependencies": {
+    "igniteui-dockmanager": "1.14.2",
+    "igniteui-react": "18.6.0",
+    "igniteui-react-core": "18.6.0",
```

Comment:
Update `igniteui-react` and `igniteui-react-core` packages to newest version **18.6.1-alpha.0** for each sample.